backtrace-library/src/main/java/backtraceio/library/BacktraceDatabase.java

```diff
@@ -177,6 +177,10 @@ public Boolean setupNativeIntegration(BacktraceBase client, BacktraceCredentials
         }
         // Path to Crashpad native handler
         String handlerPath = _applicationContext.getApplicationInfo().nativeLibraryDir + _crashpadHandlerName;
+        File handlerPathFile = new File(handlerPath);
+        if (!handlerPathFile.exists()) {
```

Comment:
there is a method called `isFileExists` in class `FileHelper` that does exactly this

Reply:
Cool, adjusted the code. thank you!